### DISCUSSION TITLE
UUID issue: error matches a different uuid with the same timestamp.

### DISCUSSION THREAD

**Discussion by GiniJo** [NightscoutFoundation/xDrip#3207] [Bug reports]
Hello,
 Have unstable connection xdrip+android 9.1 + dexcom1.
Have implemented all recommendations, but have very unstable connection (connection is lost almost every 10 or 20 minutes).

![IMG_6066](https://github.com/NightscoutFoundation/xDrip/assets/152024021/f1bf52dd-b1b7-47cb-993d-01fb48938039)

In log report getting error message BgReading: UUID issue: error matches a different uuid with the same timestamp: <…>

If you have any comments or recommendations, would appreciate !

g.

**Comment by Navid200** [comment]
Can you post a screenshot of the G5/G6/G7 status page?
If you want, you can blackout the transmitter id.

**Reply by GiniJo** [reply]
Please find few screenshots. 
![IMG_6081](https://github.com/NightscoutFoundation/xDrip/assets/152024021/54e840a7-0a49-4b7a-9217-1a100582b529)
![IMG_6072](https://github.com/NightscoutFoundation/xDrip/assets/152024021/60015ba4-56ba-40d2-80ee-72edd554c7aa)
![IMG_6075](https://github.com/NightscoutFoundation/xDrip/assets/152024021/903e9b0e-c9b7-4c15-bdea-76231de18305)
![IMG_6073](https://github.com/NightscoutFoundation/xDrip/assets/152024021/fc5e8c6f-9e5b-4985-85fb-c20fef20214e)

**Reply by GiniJo** [reply]
Additional note:currently, full android watch (with android 9.1) is used as master collector.

**Reply by Navid200** [reply]
You can have one device to collect from the mobile slot of your Dexcom One.  That can be either your phone, or your full Android watch.
If you have both enabled, it is not surprising that you don't get consistent results.

**Reply by GiniJo** [reply]
Thank you so much  for your comment. When I switched from phone to watch, I have changed my phone to follower mode, i.e. watch is set as master, and phone is set as follower. However, it looks that there is still some issue that two devices intend to collect from one source. Would appreciate if you can share your view, if it is possible somehow to check or identify this second source which request data from Dexcom One. Apology if my question sounds simply, but I am new in this field. Thank you in advance !

**Reply by Navid200** [reply]
Do all your settings on the phone, other than collection having been disabled now, match the ones shown here?
https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html
